src/container/srv_container.c

```diff
@@ -5112,18 +5381,54 @@ cont_op_with_svc(struct ds_pool_hdl *pool_hdl, struct cont_svc *svc,
 	else
 		ABT_rwlock_wrlock(svc->cs_lock);
 
+	uuid_copy(mdr_key.mdk_client_id, in->ci_cli_id);
+	mdr_key.mdk_client_time = in->ci_time;
+	mdr_val.mdv_rc = 1234567;
+	d_iov_set(&key, &mdr_key, sizeof(mdr_key));
+	d_iov_set(&val, &mdr_val, sizeof(mdr_val));
+
+#if 0
```

Comment:
Design/planning discussions have suggested we start out with an rdb-only implementation, and not implement a B+ Tree cache unless it's needed. This #if 0 is just leftover from an experiment I ran to try to look up the RPC operation in the B+ Tree.